src/matrix/room/BaseRoom.js

```diff
@@ -378,8 +381,44 @@ export class BaseRoom extends EventEmitter {
         return this._summary.data.membership;
     }
 
-    get powerLevels() {
-        return this._timeline.powerLevels;
+    async loadPowerLevels() {
+        const timelineReader = new TimelineReader({
```

Comment:
The timeline reader is not needed for the power levels. This is the part the reads the messages in the Timeline class.